### PR TITLE
Create test files

### PR DESCRIPTION
**Pull request by mariaalouisaa**
- 📝 I hope you don't mind I went ahead and created the test-helper and test files and linked them all together.
- 💻 I also put in a template for testing the UI for us to reference later. Please note - this is just a template and will need to be changed when we do testings later.
- 📚 I made a note of the three minimum tests we need to do (as stated in the project brief).
- ❌ Everything is coded **out** so that it _shouldn't_ affect the console while we're working on the JS! 
- 🤞 I copied it all from that homework we had from Oli so it should hopefully work well when we eventually have a go at it.

### CODE REVIEW

**Review by AsmahanM — APPROVED** [review]
Thanks for doing this ahead of time!

File: index.html [hunk 1]
```diff
@@ -26,7 +26,8 @@ <h1>Binge Pad</h1>
     <!-- static class attribute 
     <div class="tv-static"></div>
     -->
-
+    <script src="test-helpers.js"></script>
```
Comment (PJSalter) [inline index.html:29]: This is great, thank You for adding the script and src for the test helpers where there ready 🐰

File: test-helpers.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+function equal(actual, expected, message) {
```
Comment (PJSalter) [inline test-helpers.js:1]: Very clearly written, brilliant work 💯 😸 🤩

File: test-helpers.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  if (actual !== expected) {
+    const defaultMessage = `${expected} is different to ${actual}`;
+    console.info("Pass: " + (message || defaultMessage));
+  } else {
+    const defaultMessage = `${expected} is the same as ${actual}`;
+    console.error("Fail: " + (message || defaultMessage));
+  }
+}
+
+function test(name, testFunction) {
```
Comment (PJSalter) [inline test-helpers.js:21]: This has given us an awesome start for when we are ready on testing YAAAAAY 🏎️ 👾 ✨

File: test-helpers.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  if (actual === expected) {
+    const defaultMessage = `Expected ${expected} and received ${actual}`;
+    console.info("Pass: " + (message || defaultMessage));
+  } else {
+    const defaultMessage = `Expected ${expected} but received ${actual} instead`;
+    console.error("Fail: " + (message || defaultMessage));
+  }
+}
+
+function notEqual(actual, expected, message) {
```
Comment (PJSalter) [inline test-helpers.js:11]: notEqual function is clear and precise ⭐

File: test-helpers.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+function equal(actual, expected, message) {
+  if (actual === expected) {
+    const defaultMessage = `Expected ${expected} and received ${actual}`;
```
Comment (PJSalter) [inline test-helpers.js:3]: I enjoyed reading your equal function code 🙌

File: index-test.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
+/*
+
```
Comment (PJSalter) [inline index-test.js:2]: 💯 ✨

**Review by PJSalter — APPROVED** [review]

**Pull request merged**